CI-CD/AzureDevOps/azure-pipelines_dotnet_sbom.yml

```diff
@@ -53,5 +53,5 @@ steps:
     WS_APIKEY: $(APIKEY)
     WS_USERKEY: $(USERKEY)
 
-- publish: $(System.DefaultWorkingDirectory)/whitesource
-  artifact: Whitesource
\ No newline at end of file
+- publish: $(System.DefaultWorkingDirectory)/Mend
+  artifact: Mend
```

Comment:
I'm thinking if should split it? today with WS, it uploads the logs and the SPDX report and now it will be only the report